Diff/diff.py

```diff
@@ -63,92 +65,118 @@ def is_visible(self, files):
 
 class DiffChangesCommand(sublime_plugin.TextCommand):
 
-    def run(self, edit):
+    def run(self, edit: sublime.Edit):
+
+        view: sublime.View = self.view
+        window: typing.Optional[sublime.Window] = view.window()
```

Comment:
unnecessary annotations

also in many places below. I won't be repeating myself